### ISSUE TITLE
Drawing App using Java Script

### ISSUE THREAD

**Issue by Harikaraja** [TusharKesarwani/Front-End-Projects#630]
I would like to build a Drawing app where we can draw some shapes (like rectangle, triangle) on the available canvas with different colors , clear the canvas , and downlaod the canvas as image.

**Comment by Vanshiii203** [comment]
This project is already present in the repo

**Comment by Harikaraja** [comment]
Can I enhance it mam

**Comment by Vanshiii203** [comment]
Yes u can create an issue for enhancement in the existing project
